org.qt_project.qtcreator.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.qt_project.qtcreator
+runtime: org.kde.Sdk
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: qtcreator
+# Qt Creator already provides desktop launchers and AppData metadata
+# but since we cannot use the original AppID, we have to rename the original files
+rename-desktop-file: org.qt-project.qtcreator.desktop
+rename-appdata-file: org.qt-project.qtcreator.appdata.xml
+rename-icon: QtProject-qtcreator
+finish-args:
+# Qt Creator crashes GNOME Shell 3.36 on Wayland
+# - --socket=wayland
+# - --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --share=network
+- --device=dri
+- --filesystem=host
+cleanup:
+- /include
+- /lib/arch
+- /lib/cmake
+- /lib/libQt6Core5Compat.prl
+- /lib/metatypes
+- /lib/qtcreator/objects-RelWithDebInfo
+- /lib/x86_64-linux-gnu
+- /mkspecs
+- /modules
+- /qml
+- /share/cmake
+- /share/pkgconfig
+- "*.a"
+cleanup-commands:
+- mv /app/lib/*-linux-gnu/* /app/lib/
```

Comment:
This is wrong. Very likely that you're breaking the CMake configs of the Qt module. Look inside `/app/lib/<ARCH>-linux-gnu/cmake/Qt6*/Qt6*Targets*.cmake`  
I suggest instead to create symlinks like this
```
ln -sr ${FLATPAK_DEST}/lib/${FLATPAK_ARCH}-linux-gnu/libQt*.so* -t ${FLATPAK_DEST}/lib/
```

Reply:
Well, the building worked. So this should not be a problem. But using `FLATPAK_ARCH` is probably the better option to resolve these architecture-dependent paths when setting the Qt5 compatibility module path when building Qt Creator. I will consider this later.

Reply:
This is after the build when all the cmake files are removed. Yes, it will break the cmake script if I would do this during the build, but this does not matter at this point because the cmake script does not exist anymore after the cleanup.

I am moving the library files to `/app/lib` because the linker will not find the so files otherwise and the flatpak will just crash because `qtcreator` cannot find `libQt6Core5Compat.so.6`. I could symlink them but the effect will be the same.